lib/project-view.coffee

```diff
@@ -124,8 +138,10 @@ module.exports = ProjectView =
     # Shorten root path if possible
     userHome = fs.getHomeDirectory()
     normRootPath = path.normalize(rootPath)
-    if normRootPath.indexOf(userHome) is 0
-      # Use also tilde in case of Windows as synonym for the home folder
+    if atom.config.get 'project-view.regexMatch'
```

Comment:
I want to learn the difference.
Because a config can't be null, aren't they same?